examples/all-clusters-app/esp32/main/CMakeLists.txt

```diff
@@ -81,6 +81,7 @@ set(SRC_DIRS_LIST
                       "${CMAKE_SOURCE_DIR}/third_party/connectedhomeip/src/app/clusters/wake-on-lan-server"                      
                       "${CMAKE_SOURCE_DIR}/third_party/connectedhomeip/src/app/clusters/pump-configuration-and-control-server"
                       "${CMAKE_SOURCE_DIR}/third_party/connectedhomeip/src/app/clusters/power-source-configuration-server"
+                      "${CMAKE_SOURCE_DIR}/third_party/connectedhomeip/src/app/clusters/ota-requestor"
```

Comment:
@tima-q The support is now being reverted in #13437

Reply:
Indeed, but if I add the OTA requestor it in the normal cluster build flow - and remove the stub, it becomes a dependency thing. It's not tied into ESP32 alone, but for any build using the requestor.
OTA Requestor needs an OTA Provider client, so without the combination, the app doesn't build anymore.
I see 1 option currently, untill ZAP is able to support both client/server roles, that's making the all-clusters-app an OTA provider client + OTA requestor server only.
Any thoughts on this ?

Reply:
@tima-q, Please take a look here: https://github.com/project-chip/connectedhomeip/pull/13437#issuecomment-1009993943

Reply:
@shubhamdp Thanks for the pointer to @bzbarsky-apple 's comment.
I can remove the server here if you like to keep it contained in 1 PR

Reply:
I raised a PR #13454

Reply:
ok - thanks - will match the changes here